Data/Vector/Unboxed/V3.hs

```diff
@@ -0,0 +1,112 @@
+{-# LANGUAGE CPP                 #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE ViewPatterns        #-}
+
+-- For when DEBUG is enabled
+{-# OPTIONS_GHC -fno-warn-unused-imports #-}
```

Comment:
why? ;) I'll clean up imports/exports later, if you don't mind, rather than cover up disorder. :)